TelegramBot.py

```diff
@@ -29,11 +32,13 @@ def __init__(self, owner, name, port, counter_limit, token, chat_id, recheck_dur
         self.owner = owner
 
     def goto_state_failed(self):
-        message = "{} Your server isn't responding properly. Please check it: {}".format(self.owner, self.name)
+        message = (
+            "{} Your server isn't responding properly.\
+            Please check it: {}".format(self.owner, self.name))
```

Comment:
The parenthesis are not needed (also below in other places). Splitting the string using `\` will add the spaces you added for indentation on the next line into the resulting string. I propose the following:
```py
message = "{} Your server isn't responding properly."
message += "Please check it: {}"
message = message.format(self.owner, self.name)
```